libs/games/playground/src/Playground.ts

```diff
@@ -12,13 +12,13 @@ export class Playground extends Game {
     super({
       ...props,
       systems: [
-        ClickableSystem(props.renderer, randomName),
+        ClickableSystem(props.renderer, randomName, "isometric"),
         ControllerSystem(props.renderer, randomName),
         DebugSystem(props.renderer),
         NetcodeSystem(props.renderer, props.net, randomName),
-        PhysicsSystem(props.renderer),
+        PhysicsSystem(props.renderer, "isometric"),
         PlayerSpawnSystem(props.renderer, randomName),
-        RenderSystem(props.renderer),
+        RenderSystem(props.renderer, "isometric"),
```

Comment:
in the future, may want `isometric` and other settings be applied to `Game`, instead of individual system params